.github/workflows/label_docs.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  pull_request_target:
+    types:
+      - labeled
+      - unlabeled
+
+jobs:
+  dispatch:
+    runs-on: ubuntu-latest
+    if: github.event.label.name == 'doc needed'
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set workflowId environment variable
+        id: set_workflow_id
+        run: |
+          if [[ "${{ github.event.action }}" == "labeled" ]]; then
+            echo "workflowId=new-migrated-issue.yml" >> $GITHUB_ENV
+          else
+            echo "workflowId=delete-migrated-issue.yml" >> $GITHUB_ENV
+          fi
+
+      - name: Dispatch
+        uses: benc-uk/workflow-dispatch@v1 
+        with: 
+          workflow: ${{ env.workflowId }}
+          repo: noir-lang/docs
+          ref: main 
```

Comment:
Small nit: @kevaundray wanted to standardize all default branches to `master`

Reply:
Thats a good catch -- I've just set the default to master since most of our repositories should already use this

Reply:
I'll change the branch name to "master" on the docs repo

Reply:
done